tests/ForStatements/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -57,9 +78,23 @@ contract ForStatements {
             a++;
         }
 
-        for (; ; ) {
+        for (;;) {
             // #178
         }
+
+        for (i = 0; ; ) {}
+
+        for (i = 0; i < 100; ) {}
+
+        for (i = 0; ; i++) {}
+
+        for (i = 0; i < 100; i++) {}
+
+        for (; i < 100; i++) {}
+
+        for (; i < 100; ) {}
+
+        for (; ; i++) {}
```

Comment:
this one I mean ^^

Reply:
hehe this is based on how prettier js formats `for` statements.
When in doubt I always go to prettier js,
```Solidity
for (; ; i++) {}

for (;; i++) {}
```

The change is very easy, you can see which one you like better and we can work towards that.

Reply:
I suggest you try the different combinations and see how they appeal to your eyes. I did this based on what prettierJS does but I also tried others and just didn't like it as I got lost at a quick glance.

Reply:
so you prefer w/ a space in between; which is also what prettier-js does correct?

Reply:
I like that there's a space after the semicolon whenever there's at least one expression in the `for` statement.
Only when the `for` is empty I like it without the spaces.
This just happens to coincide with prettier-js.

Reply:
sweet, let's go w/ this one then!